contracts/stellar-example/src/tests/test.rs

```diff
@@ -32,10 +32,10 @@ struct TestConfig<'a> {
     app: ExampleClient<'a>,
 }
 
-fn setup_app<'a>(env: &Env) -> TestConfig<'a> {
+fn setup_app<'a>(env: &Env, chain_name: Option<String>) -> TestConfig<'a> {
```

Comment:
```suggestion
fn setup_app<'a>(env: &Env, chain_name: String) -> TestConfig<'a> {
```
Always require it, since a full deployment of example contract includes ITS as well